tests/StorageBackendTest.php

```diff
@@ -82,16 +42,48 @@ public function testStorageAssignBackend(string $backendName): void
             'url' => getenv('KBC_MANAGE_API_URL'),
             'token' => $token['token'],
         ]);
-        $bucketId = $sapiClient->createBucket('test', 'in');
-        $bucket = $sapiClient->getBucket($bucketId);
-        $this->assertEquals($backendName, $bucket['backend']);
 
-        $sapiClient->dropBucket($bucketId);
+        $workspace = new Workspaces($sapiClient);
+        $workspace = $workspace->createWorkspace();
 
-        $this->client->removeProjectStorageBackend($project['id'], $backendToAssign['id']);
+        try {
+            $this->client->removeStorageBackend($newBackend['id']);
+            $this->fail('should fail because backend is used in project and workspace');
+        } catch (ClientException $e) {
+            $this->assertSame(
+                sprintf(
+                    'Storage backend is still used: in project(s) with id(s) "%d" in workspace(s) with id(s) "%d". Please delete and purge these projects.',
+                    $project['id'],
+                    $workspace['id']
+                ),
+                $e->getMessage()
+            );
+        }
 
         $this->client->deleteProject($project['id']);
-        $this->client->purgeDeletedProject($project['id']);
+        $this->waitUntilProjectWillBeDeleted($project);
```

Comment:
je otázka jestli by to teda nemělo být waitForProjectPurge (což je to co to dělá IMHO) a taky jestli by to nemělo mít v parametru jen ID projektu.

Reply:
Chcel som to ale z toho `$this->client->listStorageBackend();` nejde nejak dobre urcit to podla cohoby som to mohol bezpecne mazat :/

Reply:
tak to bolo len vycuknutie uz existujucej metody, tak som to este zrefaktoroval

Reply:
jj vím